src/Common/src/Interop/Unix/System.Native/Interop.Permissions.cs

```diff
@@ -5,7 +5,7 @@
 
 internal static partial class Interop
 {
-    internal static partial class libc
+    internal static partial class Sys
     {
         [Flags]
         internal enum Permissions
```

Comment:
This is the only time I can remember where I wished we had octal in C#, so these constants could be defined the same as they are on the native side.